src/wp-includes/class-wp-theme-json.php

```diff
@@ -729,14 +730,15 @@ protected static function append_to_selector( $selector, $to_append, $position =
 	 * @return array Block metadata.
 	 */
 	protected static function get_blocks_metadata() {
-		if ( null !== static::$blocks_metadata ) {
+		$registry = WP_Block_Type_Registry::get_instance();
+		$blocks   = $registry->get_all_registered();
+
+		// Do we have metadata for all currently registered blocks?
```

Comment:
Avoid the word "we".
```suggestion
		// Is there metadata for all currently registered blocks?
```